src/processors/ipfsFile/errorProcessor.ts

```diff
@@ -29,7 +43,22 @@ export const ipfsFileErrorRetry: Processor = {
       numberOfRetries: (ipfsFile.numberOfRetries ?? 0) + 1,
       lastRetry: new Date()
     }));
-    await clients.db.upsert(Table.ipfsFiles, errorUpdates, 'url', undefined, true);
+    await clients.db.upsert(Table.ipfsFiles, errorUpdates, 'cid', undefined, true);
+  },
+  initialCursor: undefined
+}
+
+export const ipfsFileUrlErrorRetry: Processor = {
+  name: `ipfsFileUrlErrorRetry`,
+  trigger: ipfsFilesUrlsWithErrors,
+  processorFunction: async (input: IPFSFileUrl[], clients: Clients) => {
+    const errorUpdates: IPFSFileUrl[] = input.map((ipfsFileUrl) => ({
+      ...ipfsFileUrl,
```

Comment:
do we need all the fields with ...? can't we just set the primary key, ie, url and do an update?